src/components/GaugeCard/GaugeCard.story.jsx

```diff
@@ -69,6 +69,35 @@ storiesOf('Watson IoT Experimental/GaugeCard', module)
       </div>
     );
   })
+  .add('basic with expand', () => {
```

Comment:
It might be helpful to add something into the expanded content as an example as to something that would be there. It looks very weird being so empty, and the arrow seems to go off center.

![Screen Shot 2020-09-12 at 10 03 22 AM](https://user-images.githubusercontent.com/25177649/92998392-35758500-f4df-11ea-99bb-a009ce7fac79.png)

Reply:
Fixed this by removing the expand update, this is to keep it in line with the other expand stories